app/decorators/spree/line_item_decorator.rb

```diff
@@ -1,74 +1,72 @@
+    base.has_many :product_customizations, dependent: :destroy
 
-      base.validate :product_customization_should_exist_if_required
-    end
+    base.validate :product_customization_should_exist_if_required
+  end
 
-    def product_customization_should_exist_if_required
-      existing_product_customization_types = product_customizations.map(&:product_customization_type).uniq
-      product.product_customization_types.each do |product_customization_type|
-        if product_customization_type.is_required? &&
-           !existing_product_customization_types.include?(product_customization_type)
-          errors.add(:order, "missing required product customization: #{product_customization_type.name.capitalize}")
-        end
+  def product_customization_should_exist_if_required
+    existing_product_customization_types = product_customizations.map(&:product_customization_type).uniq
+    product.product_customization_types.each do |product_customization_type|
+      if product_customization_type.is_required? &&
+          !existing_product_customization_types.include?(product_customization_type)
+        errors.add(:order, "missing required product customization: #{product_customization_type.name.capitalize}")
       end
     end
+  end
 
-    def options_text # REFACTOR
-      if customized?
-        str = Array.new
-
-        ad_hoc_opt_values = ad_hoc_option_values.sort_by(&:position)
-        ad_hoc_option_values.sort_by(&:position).each do |pov|
-          str << "#{pov.option_value.option_type.presentation} = #{pov.option_value.presentation}"
-        end
+  def options_text # REFACTOR
+    if customized?
+      str = Array.new
 
-        product_customizations.each do |customization|
-          price_adjustment = (customization.price == 0) ? "" : " (#{Spree::Money.new(customization.price).to_s})"
-          customization_type_text = "#{customization.product_customization_type.presentation}#{price_adjustment}"
-          opts_text = customization.customized_product_options.map(&:display_text).join(', ')
-          str << customization_type_text + ": #{opts_text}"
-        end
+      ad_hoc_opt_values = ad_hoc_option_values.sort_by(&:position)
+      ad_hoc_option_values.sort_by(&:position).each do |pov|
+        str << "#{pov.option_value.option_type.presentation} = #{pov.option_value.presentation}"
+      end
 
-        str.join('\n')
-      else
-        variant.options_text
+      product_customizations.each do |customization|
+        price_adjustment = (customization.price == 0) ? "" : " (#{Spree::Money.new(customization.price).to_s})"
+        customization_type_text = "#{customization.product_customization_type.presentation}#{price_adjustment}"
+        opts_text = customization.customized_product_options.map(&:display_text).join(', ')
+        str << customization_type_text + ": #{opts_text}"
       end
-    end
 
-    def customized?
-      product_customizations.present? || ad_hoc_option_values.present?
+      str.join('\n')
```

Comment:
uhh, this isn't going to be treated as a newline when wrapped in single quotes...

e.g.
```ruby
irb(main):003:0> puts ["1", "2"].join('\n')
1\n2
=> nil
irb(main):004:0> puts ["1", "2"].join("\n")
1
2
=> nil
```